src/lib/gridsterDraggable.service.ts

```diff
@@ -243,9 +245,25 @@ export class GridsterDraggable {
       this.mousedown();
       this.touchstart();
     }
+    if (!enableDrag) {
+      this.dragStartFunction = this.onMouseDown.bind(this);
+      this.mousedown = this.gridsterItem.renderer.listen(this.gridsterItem.el, 'mousedown', this.dragStartFunction);
+      this.touchstart = this.gridsterItem.renderer.listen(this.gridsterItem.el, 'touchstart', this.dragStartFunction);
+    }
+  }
+
+  onMouseDown() {
+    if (this.gridsterItem.item.isSelected) {
+      this.selection.unSelectItem.emit(this.gridsterItem);
+    } else {
+      this.selection.selectItem.emit(this.gridsterItem);
+    }
+    this.gridster.updateGrid();
```

Comment:
why need to update grid on selection?
This method only updates the lines grid from the background.
Yeah name is not the best for this method.